djangocms_snippet/models.py

```diff
@@ -106,6 +106,10 @@ class SnippetPtr(CMSPlugin):
 
     search_fields = ['snippet__html'] if SEARCH_ENABLED else []
 
+    def get_short_description(self):
+        snippet_label = SnippetGrouper.objects.filter(pk=self.snippet_grouper.pk).first()
```

Comment:
There is a weird edge case where .first() can return different things. May be worth ordering this by pk!